src/draw.rs

```diff
@@ -67,3 +72,70 @@ where
         }
     }
 }
+
+pub struct RoundedRect {
+    radius: Radius,
+    color: Color,
+}
+
+impl RoundedRect {
+    fn new(radius: Radius, color: Color) -> Self {
+        Self { radius, color }
+    }
+}
+
+impl Drawable for RoundedRect {
+    fn draw(self, dt: &mut DrawTarget<'_>, scale: u16, space: Space, start_point: Point) -> Space {
+        let Point { x, y, _unit } = start_point;
+        let Space { width, height } = space;
+
+        let max_radius = width.min(height) / 2.0;
```

Comment:
This one looks like an artificial restriction. Why is that needed?

Reply:
The idea here is to employ a similar principle as for the CSS border-radius property: We don't want any corner curves to overlap and thus cap the radius to the above described one. This results in semi-circles at the smaller side for too large radii (like for the default input text field).

While this is not the exact same behavior as for CSS ([it specifies to resizes all radii proportionally](https://www.w3.org/TR/css-backgrounds-3/#corner-overlap)), it should be close enough and allows specifying large values to force maximum rounding (to semi-circles).

Please let me know if I should change the implementation to the one specified by CSS.

Reply:
Oh, that's interesting. I don't think the complication of proportional resizing really worth it so let it be like this. But a small comment here might be helpful there

Reply:
Alright, thank you!